src/Orchard.Web/Modules/Orchard.Roles/Activities/UnassignRoleActivity.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Orchard.ContentManagement;
+using Orchard.Data;
+using Orchard.Environment.Extensions;
+using Orchard.Localization;
+using Orchard.Logging;
+using Orchard.Roles.Models;
+using Orchard.Roles.Services;
+using Orchard.Workflows.Models;
+using Orchard.Workflows.Services;
+
+namespace Orchard.Roles.Activities {
+    [OrchardFeature("Orchard.Roles.Workflows")]
+    public class UnssignRoleActivity : Task {
+        private readonly IWorkContextAccessor _workContextAccessor;
+        private readonly IRepository<UserRolesPartRecord> _repository;
+        private readonly IRoleService _roleService;
+
+        public UnssignRoleActivity(
+            IWorkContextAccessor workContextAccessor,
+            IRepository<UserRolesPartRecord> repository,
+            IRoleService roleService) {
+            _workContextAccessor = workContextAccessor;
+            _repository = repository;
+            _roleService = roleService;
+            T = NullLocalizer.Instance;
+            Logger = NullLogger.Instance;
+        }
+
+        public Localizer T { get; set; }
+        public ILogger Logger { get; set; }
+
+        public override string Name
+        {
```

Comment:
It works fine for me, maybe a setting, or another plugin taking over (Resharper?)